app/Http/Controllers/api/BalanceEntryController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers\api;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+use App\BalanceEntry;
+
+use Illuminate\Support\Facades\DB;
+
+class BalanceEntryController extends Controller
+{
+    public function index(Request $request)
+    {
+        $result = BalanceEntry::all()->sortByDesc('date')->values()->all();
```

Comment:
Doing `orderBy` in the query would be more efficient that `sortBy` on the collection.